Assignment5/Assignment5_QNo2/SolAssignmentNo5_QNo2.rb

```diff
@@ -0,0 +1,21 @@
+# => 2. Write a program to define a method inside a
+# => module and then use the module in such a way that
+# => the method can be accessible as a class method of that class.
+
+# => import the two module file
+require_relative 'module_class_method.rb'
+
+class ClassModuleInterface
+  # => extend module to migrate to class method
+  include ModuleDemoClassMethod
+
+  def self.call_class_method
+    puts "The addition from Module Add is = "
+    ModuleDemoClassMethod.behave_class_method
+  end
```

Comment:
@HrisikeshSinghaChowdhury without using a explicit class method how will use use that module method as class method, please and example of that to access directly

Reply:
done